application/comit_node/tests/rfc003_ether_htlc.rs

```diff
@@ -10,12 +10,15 @@ pub mod ethereum_wallet;
 pub mod htlc_harness;
 pub mod parity_client;
 
-use crate::htlc_harness::{ether_harness, EtherHarnessParams, HTLC_TIMEOUT, SECRET};
+use crate::htlc_harness::{
+    ether_harness, CustomSizeSecret, EtherHarnessParams, HTLC_TIMEOUT, SECRET,
+};
 use ethereum_support::{Bytes, EtherQuantity, H256, U256};
 use spectral::prelude::*;
 use testcontainers::clients::Cli;
 
-const HTLC_GAS_COST: u64 = 10885000;
+const HTLC_GAS_COST: u64 = 11039400;
+const HTLC_GAS_COST_SHORT_SECRET: u64 = 11033000;
```

Comment:
❓  Testing with these seems pretty brittle. Can't we just test that the balance on the HTLC hasn't changed with a short secret?

Reply:
Ah yes indeed, this is an artefact from when we were testing that the security bug existed